kotlin/query-with-cql-filters/README.metadata.json

```diff
@@ -29,7 +29,7 @@
         "TimeExtent"
     ],
     "snippets": [
-        "src/main/java/com/esri/arcgisruntime/sample/query_with_cql_filters/MainActivity.kt"
+        "src/main/java/com/esri/arcgisruntime/sample/querywithcqlfilters/MainActivity.kt"
```

Comment:
didn't this change already get merged in a previous PR? Just checking here just in case something has reverted :)